WordPress/gutenberg
# Make `CircularOptionPicker` focus styles resilient to button size changes

## What?

As per #54156, this PR adjusts the `CircularOptionPicker` focus ring style so as not to be hardcoded to a specific size.


## Why?

`CircularOptionPicker` focus styles don't account for different option sizes, as they are hard-coded to specific dimensions. This means that any variation in option size isn't accounted for.


## How?

This patch updates how the focus ring is styled to make it adaptable for differing control sizes.


## Testing Instructions

https://github.com/WordPress/gutenberg/assets/159848/0a4f2554-f77e-4606-bb9e-5fcfaae5bab6

Navigate to the editor styles colour palette:
 1. Appearance
 2. Editor
 3. Styles
 4. Edit styles...
 5. Colours
 6. Palette

Using the keyboard, tab through the various controls until the colour swatch options are highlighted. The focus ring should wrap the swatch, and not look off.

## Screenshots or screencast

### Before
![Five small colour swatches, with labels; one has focus, but the focus ring is offset and too large.](/WordPress/gutenberg/assets/159848/230617ea-13e1-48a3-af67-0ecd8746bb3e)

### After
![Five small colour swatches, with labels; one has focus, as indicated by a ring around it.](/WordPress/gutenberg/assets/159848/548ddeb7-33b6-46ef-afd9-0f741372c3fa)
